index.js

```diff
@@ -7,7 +7,8 @@ const Hyperswarm = require('hyperswarm')
 const ReadyResource = require('ready-resource')
 const z32 = require('z32')
 const b4a = require('b4a')
-const { Router, dispatch } = require('./spec/hyperdispatch')
+const { Router, encode } = require('./spec/hyperdispatch')
+const BlindPeering = require('@holepunchto/blind-peering')
```

Comment:
just noting that we should publish before merging, but i think we are basically ready on that